docs/platform/howto/project-support-center.rst

```diff
@@ -1,24 +1,69 @@
 
-In addition to the Basic support tier, we offer three additional tiers that offer faster response times, phone support, and other services. For more information about support plans, please see `Aiven support details page <https://aiven.io/support-services>`_ or contact sales@Aiven.io to discuss the paid support options.
+The Aiven service level agreement (SLA) is available on `the SLA page <https://aiven.io/sla>`_. Custom SLAs are available for premium plans. Contact the sales team at sales@Aiven.io for more details.
+
+
+.. _upgrade-support-tier:
+
+Upgrade your support tier
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you have questions or want to downgrade your support tier, contact your account team. 
+
+To upgrade your organization's support tier in the `Aiven Console <https://console.aiven.io/>`_:
+
+#. In the organization, click **Support**.
+
+#. In the **Current support tier** section, click **Upgrade to Enterprise**. 
+
+#. Choose the support tier you want to upgrade to and click **Choose tier**. 
+
+#. Select a **Start date**. 
+    
+   .. note::
+    If you select the current month, you will be charged a percentage of the total service spend for the whole month, starting from the 1st.
+
+#. Select a **Billing group**. 
+
+   .. important::
+    The support costs for all current and future services in the selected organization and all of its organizational units will be added to the invoice for this billing group.
+
+#. Click **Upgrade tier**.
+
+It typically takes 1-2 business days to set up the new support tier. You can view the status of your request on the support page under **Current support tier**.
 
-You can create and track support tickets in the `Aiven Console <https://console.aiven.io/>`_ following the instructions on this page. For other services included in your support tier - such as business reviews or disaster recovery planning - contact the sales team at sales@Aiven.io.
 
 Create a support ticket
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-To create support tickets in the `Aiven Console <https://console.aiven.io/>`_:
+You can create and track support tickets in the Aiven Support Center. For other services included in your support tier - such as business reviews or disaster recovery planning - contact the sales team at sales@Aiven.io.
+
+#. In the `Aiven Console <https://console.aiven.io/>`_, click **Support**.
 
-#. Click **Support**.
+#. Click **Go to Aiven Support Center**.
 
 #. Click **Create ticket**. 
 
-#. Enter a **Subject** and detailed **Description** of the issue. 
+#. Enter email addresses to CC in the support ticket. All new comments and updates will be sent to these emails.
+
+#. Enter a **Subject**.
+
+#. Select a **Severity** level:
+
+   * Low: The primary functions are working, but some secondary functionality is not working.
+   * High: The primary functions are working, but severely limited or slow.
+   * Critical: The primary functions are not working and it's not possible to find workarounds.
```

Comment:
maybe let's use the "official" description from https://aiven.io/support-services?